src/core/primitives/Select/fragments/SelectPrimitiveRoot.tsx

```
@@ -59,30 +59,23 @@
 
-        const floatingElement = refs.floating.current;
-        if (!floatingElement) return;
-        console.log(selectedIndex)
-        const position = (selectedIndex + 1) * (floatingElement.clientHeight / floatingElement.children.length);
-        setOffsetPositionValue(-position);
-    }, [isOpen, selectedIndex, refs.floating]);
+            const relativeTop = rectA.top - rectB.bottom;
+            setOffsetPositionValue(relativeTop);
+        }
+    }, [refs.floating.current, selectedItemRef.current]);
+    }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)